Source/WebCore/Modules/mediasource/MediaSource.cpp

```diff
@@ -230,6 +230,7 @@ MediaSource::MediaSource(ScriptExecutionContext& context, MediaSourceInit&& opti
 #if !RELEASE_LOG_DISABLED
     , m_logger(logger(context))
 #endif
+    , m_private(nullptr)
```

Comment:
Explicitly setting a RefPtr to nullptr is not right.

Reply:
Removed.